src/constants/propTypes/patientPropTypes.js

```diff
@@ -25,14 +26,11 @@ const Patient = PropTypes.shape({
   weight: PropTypes.number,
   height: PropTypes.number,
   bmi: PropTypes.number,
-  diagnosis: PropTypes.arrayOf(PropTypes.string),
   pesel: PropTypes.string,
   date_of_birth: PropTypes.string.isRequired,
   note: PropTypes.string,
-  person_authorized: PropTypes.string,
-  phone_authorized: PropTypes.string,
-  guardianship: PropTypes.bool,
-  code: CodeProp,
+  person_guard: PropTypes.string,
+  phone_guard: PropTypes.string,
```

Comment:
To samo co w komentarzu ze zniknięciem patient.code. Czy w takim razie chcesz wyrzucić z ekranu AddPatientScreen pole "Kod rozpoznania"?